lib/src/gregorian_calendar.dart

```diff
@@ -98,7 +59,7 @@ class GregorianCalendar implements Calendar {
 
   /// The number of days in the month of this date
   int get monthLength {
-    final int n = _daysPerMonth[_month];
+    final int n = daysPerMonth[_month];
```

Comment:
This seems bad to make public.  It will be very easy to misuse.

Reply:
I've only made it public to the rest of the library, so that gregorian_calendar.dart and gregorian_calendar_duration.dart could share constants. I intentionally did not add the new gregorian_consts.dart file I created to the export list for that very reason.

Do you have an alternate suggestion to allow sharing of constants between the two files?

Reply:
Ahh I see now.  That's ok then.